namespace-provisioner/ootb-supply-chain.md

```diff
@@ -137,32 +160,53 @@ Using GitOps
 
 - `Pipeline:` defines how to run the tests on the source code pulled by the supply chain and which image to use that has the tools to run those tests.
 - `ScanTemplate`: defines how to run a scan, you can change how the scan is run, either for images or source code. 
 
-  A ScanTemplate defines the PodTemplateSpec used by a Job to run a particular scan (image or source). When the supply chain initiates an ImageScan or SourceScan, they reference these templates which must be in the same namespace as the workload.
+  - A ScanTemplate defines the PodTemplateSpec used by a Job to run a particular scan (image or source). When the supply chain initiates an ImageScan or SourceScan, they reference these templates which must be in the same namespace as the workload.
 
-  Although you can customize the templates, VMware recommends that you follow what is provided in the installation of the `grype.scanning.apps.tanzu.vmware.com` package. This is automatically created in all the namespaces managed by Namespace Provisioner. For more information, see [About Source and Image Scans](../scst-scan/explanation.hbs.md#about-src-and-image-scans).
+  - Although you can customize the templates, VMware recommends that you follow what is provided in the installation of the `grype.scanning.apps.tanzu.vmware.com` package. This is automatically created in all the namespaces managed by Namespace Provisioner. For more information, see [About Source and Image Scans](../scst-scan/explanation.hbs.md#about-src-and-image-scans).
 
-- `ScanPolicy`: define how to evaluate whether the artifacts scanned are compliant.
-- When an ImageScan or a SourceScan is created to run a scan, they reference a policy, the policy name must match the following [sample ScanPolicy](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/ns-provisioner-samples/testing-scanning-supplychain/scanpolicy-grype.yaml). See [Writing Policy Templates](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/scst-scan-policies.html).
+- `ScanPolicy`: define how to evaluate whether the artifacts scanned are compliant. For example, allowing one to be either very strict, or restrictive about particular vulnerabilities found.
+  - When an ImageScan or a SourceScan is created to run a scan, they reference a policy, the policy name must match the following [sample ScanPolicy](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/ns-provisioner-samples/testing-scanning-supplychain/scanpolicy-grype.yaml). 
+  - See [Writing Policy Templates](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/scst-scan-policies.html).
```

Comment:
I think this should point to the file in the repo itself, not to the staging docs

Reply:
`../scst-scan/policies.hbs.md#writing-a-policy-template`

Reply:
Updated the link